xds/src/main/java/io/grpc/xds/internal/rbac/engine/AuthorizationEngine.java

```diff
@@ -119,10 +119,8 @@ public AuthorizationEngine(ImmutableList<RBAC> rbacPolicies) throws IllegalArgum
    * Determines whether a gRPC call is allowed, denied, or unable to decide.
    * @param args evaluate argument that is used to evaluate the RBAC conditions.
    * @return an AuthorizationDecision generated by CEL Evaluation Engine.
-   * @throws InterpreterException if something goes wrong in CEL library.
    */
-  public AuthorizationDecision evaluate(EvaluateArgs<ReqT, RespT> args) 
-      throws IllegalArgumentException {
+  public AuthorizationDecision evaluate(EvaluateArgs<ReqT, RespT> args) {
```

Comment:
Just curious - why do we give up on throwing an exception here?

Reply:
`IllegalArgumentException` is a RuntimeException do adding that to the `throws` clause doesn't achieve anything?

Reply:
Because we changed `List<RbacEngine>` to `RbacEngine denyEngine` and `RbacEngine allowEngine`.